Source/WTF/wtf/text/StringView.h

```diff
@@ -71,8 +71,7 @@ class StringView final {
     StringView(std::span<const char> span LIFETIME_BOUND); // FIXME: Consider dropping this overload. Callers should pass LChars/UChars instead.
     StringView(const void* string LIFETIME_BOUND, unsigned length, bool is8bit);
     StringView(ASCIILiteral);
-
-    ALWAYS_INLINE static StringView fromLatin1(const char* characters) { return StringView { characters }; }
+    StringView(NullTerminated);
```

Comment:
This is a new implicitly Latin-1 character set constructor but the old code had a `FIXME` about removing the one we already had and kept the one most like this, which took a `const char*`, private.